### PR TITLE
config: use 9p as default shared filesystem for nemu

### PR DESCRIPTION
**Pull request by chavafg**
currently virtiofs support is being reworked to use qemu
and kernel from virtio-fs gitlab repo. The current support
that we have for virtio-fs has been disabled as required
patches for the 4.19 kernel were removed in [1].

[1] https://github.com/kata-containers/packaging/pull/703

Fixes: #2048.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -169,7 +169,7 @@ DEFENTROPYSOURCE := /dev/urandom
 
 DEFDISABLEBLOCK := false
 DEFSHAREDFS := virtio-9p
-DEFSHAREDFS_NEMU := virtio-fs
+DEFSHAREDFS_NEMU := virtio-9p
```
Comment (GabyCT) [inline Makefile]: @chavafg so it seems that this https://github.com/kata-containers/runtime/pull/2054/files#diff-ebce02844da8831ba833219d9a0d0d36R103 will be affected by this change, why we do not use `DEFSHAREDFS` at the configuration.toml just to not have the same variable with the same value

Comment (chavafg) [reply Makefile]: I could use `DEFSHAREDFS`, but I think `DEFSHAREDFS_NEMU` should only be used by `configuration-nemu.toml`. nemu related variables will be removed once we deprecate nemu, so I think we should use another variable for `virtio-fs`.
@jodh-intel @egernst any opinion on this?

**Comment by chavafg** [comment]
@GabyCT ptal, I added a new variable `DEFSHAREDFS_QEMU_VIRTIOFS` for the `configuration-qemu-virtiofs.toml`, so that it doesn't affect the nemu change.

**Pull request merged**